### PR TITLE
Let asm/disasm support more architectures

### PR DESCRIPTION
**Pull request by david942j**
Architectures added:
`aarch64 arm mips mips64 powerpc64 sparc sparc64 thumb`

and `powerpc` added for asm only.

---
This change is [Reviewable](https://reviewable.io/reviews/peter50216/pwntools-ruby/150)

### CODE REVIEW

File: test/asm_test.rb [hunk 1]
```diff
@@ -168,7 +158,7 @@ def test_make_elf
   # this test can be removed after method +run_shellcode+ being implemented
   def test_make_elf_and_run
     # run the ELF we created to make sure it works.
-    linux_only
+    linux_only('ELF can only be executed on Linux')
```
Comment (peter50216) [inline test/asm_test.rb]: No ()?
(There's one linux_only without () below)
```suggestion
    linux_only 'ELF can only be executed on Linux'
```

Comment (david942j) [reply test/asm_test.rb]: wow the feature of `suggested change` is great

**Comment by peter50216** [comment]
:lgtm:

**Comment by peter50216** [comment]
:lgtm:

**Comment by peter50216** [comment]
:+1:

**Pull request merged**